CRAWLR.xcodeproj/project.pbxproj

```diff
@@ -8,11 +8,15 @@
 
 /* Begin PBXBuildFile section */
 		1E5AD0C4C3B171AAB7B0AD29 /* Pods_CRAWLR.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 543B5F630308AB085FD4B543 /* Pods_CRAWLR.framework */; };
+		59266FA624526669003640B2 /* SearchCell.swift in Sources */ = {isa = PBXBuildFile; fileRef = 59266FA524526669003640B2 /* SearchCell.swift */; };
+		59266FA824536603003640B2 /* CustomNavController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 59266FA724536603003640B2 /* CustomNavController.swift */; };
```

Comment:
```suggestion
		59266FA824536603003640B2 /* CustomNavController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 59266FA724536603003640B2 /* CustomNavController.swift */; };
				59266FA724536603003640B2 /* CustomNavController.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = CustomNavController.swift; sourceTree = "<group>"; };
								59266FA724536603003640B2 /* CustomNavController.swift */,
												59266FA824536603003640B2 /* CustomNavController.swift in Sources */,

```
Remove the custom nav controller, as far as I can tell it's not being used. Try not to push useless code